### ISSUE TITLE
Not firing when using just gtags.js - Global Site Tag

### ISSUE THREAD

**Issue by bobbonomo** [robflaherty/scroll-depth#123]
This is just a FYI for anyone using this:

Not sure how to put this but Scroll Depth does not fire when using just Global Site Tag (gtag.js) and not within GTM which is another animal. From Goggle docs "GTM doesn't use Gtag.js it creates universal analytics tags".

Got an email from Google to implement gtag.js using the snippet code in my account. So I did what the boss asked.

Using the latest master jquery.scrolldepth.js v1.2.0 dated 2020-07-03

The mock-up in the test folder fires well using either ga or gtag or even the other 2 "fake" functions and all the options but then load the gtag.js library from Google and it does not fire.

Using the NOT minimized version I put in a console log as the first line in the sendEvent function to see what is being passed. Looked OK.

Then I threw in a console log in all the if's and nada. So in desperation I did the same in the standardEventHandler if statement and that fired. What? I don't have an event handler declared so this firing ends up going nowhere.

From the docs:
"GtmOverride:
By default, if Scroll Depth detects Google Tag Manager it assumes that you're using it for your GA implementation. If you're using GTM but not using it for GA then you can set this option to true and Scroll Depth will ignore GTM."

Somehow Scroll Depth must detect Google Tag Manager when just using gtag.js and I suspect datalayer. It never occurred to me to override something I was not using but this is what makes jquery.scrolldepth.js v1.2.0 work. Setting GtmOverride to true makes sure the whole standardEventHandler if-statement never resolves to true. [line 90]

The docs should mentioned this. I guess not many people use gtag.js and not use GTM for GA implementation. From what I read if you use GTM you don't really need Scroll Depth. 

From the WP Scroll Depth plugin:
Update: Google Tag Manager now includes Scroll Depth and Element Visibility triggers. If you are running GTM, you don’t really need this plugin.

**Comment by robflaherty** [comment]
Yes, GTM provides similar scroll depth tracking natively now so if you're using GTM you don't need this.

I don't quite follow your description of the issue but it's possible there's a bug re: gtag and GTM.

FYI this project is not under active development anymore and I'll soon be directing everyone to a new version that's currently in beta: https://github.com/robflaherty/scroll-depth-2

**Comment by bobbonomo** [comment]
Like I said I did not know how to describe it. It's all moot now that there is a new version. The best way to see the effect is to run your test/index.html with no parameters defined (defaults) then turn off both userTiming and pixelDepth and run it again. Percentage fires just once. OK I said does not fire.

Testing on a live server and loading my gtags.js snippet in <head> I saw analytics.js being called. Who did that? Not me. Not you. Must be Google. Maybe that is the problem. Maybe the problem is me.

As for my testing, it seems putting a debug statement log("sendEvent logic"); on line 102 --function sendEvent()-- of jquery.scrolldepth.js changes the results of the firing.

I've now lost motivaation for playing with version 1.2 so I will concentrate on version 2...  and joy! there no jquery to load.
